jobs/director/templates/director.yml.erb.erb

```diff
@@ -189,6 +189,12 @@ if p('blobstore.provider') == "s3"
    params['blobstore']['options']['s3cli_path'] = "/var/vcap/packages/s3cli/bin/s3cli"
 end
 
+if p('blobstore.provider') == "gcs"
+   params['blobstore']['provider'] = "gcscli"
+   params['blobstore']['options']['gcscli_config_path'] = "/var/vcap/data/tmp/director"
+   params['blobstore']['options']['gcscli_path'] = "/var/vcap/packages/bosh-gcscli/bin/bosh-gcscli"
```

Comment:
Yes, the packages folder. That's one of the last things to do as it involves Pivotal people pushing to a blobstore.